tests/foreman/api/test_organization.py

```diff
@@ -349,9 +349,9 @@ def test_positive_add_smart_proxy(self):
         @CaseLevel: Integration
         """
         # Every Satellite has a built-in smart proxy, so let's find it
-        smart_proxies = entities.SmartProxy().search()
-        self.assertGreater(len(smart_proxies), 0)
```

Comment:
@abalakh ,
I've actually started to fight with improper assertions here https://github.com/SatelliteQE/robottelo/issues/4041
:)